samples/databus/custom-serializer/Core_6/Shared/MessageWithLargePayload.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using NServiceBus;
+
+//the data bus is allowed to clean up transmitted properties older than the TTBR
+[TimeToBeReceived("00:01:00")]
```

Comment:
Copied from `FileShareDataBus` sample. Isn't TTBR required for a message containing a DataBus property? I can drop the comment because it's irrelevant to this topic.